### DISCUSSION TITLE
Help with ''raw hls''

### DISCUSSION THREAD

**Discussion by MaddisonBice** [aclap-dev/video-downloadhelper#1539] [Help and Q&A]
### Your question

I'm getting this error when i try to download videos from any site. I have been using the same website to download for a long time now so it's only ''all of a sudden''. Please advise on how i can fix this please :)
![Screenshot 2024-12-13 224401](https://github.com/user-attachments/assets/06d03655-835b-4168-a0da-972c477462c7)

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "used_history_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "record_download_history": true
}

**Comment by jcfrog** [comment]
hi,
sorry I must be missing something, I don't see any error on your screenshot.

**Comment by MaddisonBice** [comment]
When I click download it comes up with an error to report or it only downloads 10-20 minutes of whole movies. A lot of the time it just doesn’t download at all too. Thank you for taking time to look at my post

On 13 Dec 2024, at 23:51, jcfrog ***@***.***> wrote:

﻿

hi,
sorry I must be missing something, I don't see any error on your screenshot.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

My environment is Windows 11 Pro 64-bit version 24H2 build 26100.2605, Firefox 133.0.3 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

For future reference, when you give an example in a post here, you should make part of the text of your post be the URL of your example.  That makes it easy for others to help you.  At least your image shows the URL of your example but I had to manually type that in.  That is a process that is prone to errors & somebody might think you provided a URL that doesn't work when the problem was simply a transcription error, a fat-fingering on my part.  You should have done this:

https://seriesonlinehd.net/watch-movie/heretic-116767.10793839

Also, since you are communicating via E-mail here, you should do the polite thing & make sure there is no quoted material in your response before you send it.  Delete whatever is being quoted.  Perhaps your E-mail client is automatically quoting something that you never noticed.  You need to pay closer attention.  Delete whatever is being autoquoted.  You should also know that when you E-mail a post to GitHub, GitHub removes all attachments.  For example if you want to post an image, like I'm doing in this post, you can't do that via E-mail.  You have to come here via your browser, sign on, & compose your post in the web page.

In any case, I don't know why you mention Raw HLS.  Your image does not show Raw HLS.  It shows HLS, which should be handled quite easily by VDH.  When I went to your example movie, I got this:

![#01](https://github.com/user-attachments/assets/10a31715-0314-4771-9af7-7d09dbc0e34d)

That's pretty much the same as what you got.  I started the download & it looked like this:

![#02](https://github.com/user-attachments/assets/6c7ff008-b99d-4ecb-8053-53fd52b680ba)

That is quite a slow download speed.  Your web site is severely throttling its service.  That image shows a download speed of just over 1M.  But as I type this, it's been running for about 15 minutes, it's only about 1/3 of the way done, & the speed has dropped to something on the order of 600K.  Putrid speed.  But it is continuing & I will let it run.  However, it's going to take a rather long time to complete so I decided to post early.

If I do not post again, assume that my download completed successfully without incident.  Assume that the movie played fine in VLC.  I do not plan to watch it all the way through, just sample it at intervals to make sure it's all there.

You say sometimes it downloads only part of the movie.  I speculate that since this site gives such reprehensibly bad service, your downloads tend to last rather long times.  That gives things a greater chance to fall victim to line errors.  Speculate.  I can't be sure.  I'm not watching your downloads over your shoulder.  But when I have observed premature termination of downloads on my system, it is usually due to some sort of line error.

As for the download simply failing, clearly I'm not experiencing that.  But then, I'm using Firefox while you're using Chrome.  This would not be the first time Firefox handled a case better than Chrome.  In addition to the browser itself being an issue, I am using a different VDH than you are.  Since I'm using Firefox, I have access to the VDH beta versions.  It could be that there's some bug that has been fixed in the beta that has not made it into the general availability release yet.  Maybe.  I'm just guessing.  Still, you might try doing this with Firefox & the VDH beta to see if your experience improves.

**Reply by GrampaWildWilly** [reply]
Another issue is that since you are using Chrome & you do not have a license, VDH limits your HLS downloads to 1 every 2 hours.  There is no such limit on unlicensed VDH for Firefox.
